Migrate proximity to sensor entities (#29223)
home-assistant/home-assistant.github.io: source/_integrations/proximity.markdown

- Increase thermostat temperature as you near home
- Decrease temperature the further away from home you travel

## Sensors

> The following sensor entities will be created

Reviewer: ```suggestion
The following sensor entities will be created.
```

### Distance

> For each tracked device or person, the distance from the monitored zone in an unit depending on your [Home Assistant Unit System](/docs/configuration/basic) selection.

Reviewer: ```suggestion
For each tracked device or person, the distance from the monitored zone in a unit depending on your [Home Assistant Unit System](/docs/configuration/basic) selection.
```

Reviewer: It's weird to not have a verb in these sentences. Not sure what would be better.

Author: ```suggestion
For each tracked device or person, a sensor showing the distance from the monitored zone in a unit depending on your [Home Assistant Unit System](/docs/configuration/basic) selection is created.
```

what about this one?

Reviewer: Yes, that's good. But replace an with a.

You can use the [Min/Max](/integrations/min_max) integration to determine the nearest and furthest distance.

### Direction of travel

> For each tracked device or person, the direction of travel to or from the monitored zone. Values are:

Author: ```suggestion
For each tracked device or person, a Sensor showing the direction of travel to or from the monitored zone is created. Possible states are:
```

> For each tracked device or person, a Sensor showing the direction of travel to or from the monitored zone is created. Possible states are:

Author: ```suggestion
For each tracked device or person, a sensor showing the direction of travel to or from the monitored zone is created. Possible states are:
```

- `not_set`
- `arrived`
- `towards`
- `away_from`

> - `unknown`

Reviewer: The order here seems a bit awkward.

Did we keep `not_set`?

Author: you're right - we have removed `not_set` 👍

> - `not_set`
> - `arrived`
> - `towards`
> - `away_from`
> - `unknown`
> - `stationary`

Author: ```suggestion
- `arrived`
- `away_from`
- `stationary`
- `towards`
- `unknown`
```

### Nearest device

> The device or person which is nearest to the zone

Author: ```suggestion
A Sensor showing the device or person which is nearest to the zone is created.
```

> A Sensor showing the device or person which is nearest to the zone is created.

Author: ```suggestion
A sensor showing the device or person which is nearest to the monitored zone is created.
```
